index.js

```diff
@@ -72,15 +58,14 @@ const jobs = {
 }
 
 async function onEvent(event, { jobs, global }) {
-    let response = null
     if (!global.eventAndNumberMap[event.event]) {
         return null
     } else {
         const request = {
             eventName: event.event,
         }
-        response = await jobs.sendMessageWithTwilio(request).runNow()
     }
+    const response = await jobs.sendMessageWithTwilio(request).runNow()
 
     return response
```

Comment:
we shouldn't return anything, and this response is useless.
```suggestion
await jobs.sendMessageWithTwilio(request).runNow()
```